contents/blog/mobile-session-replay.md

```diff
@@ -0,0 +1,140 @@
+
+## Solving mobile replays big problems
+
+So how did we solve some of the big problems we identified earlier? Two were relatively simple:
+
+1. **Multiple platforms:** Do the work to develop mobile replay for all the platforms (which is still ongoing).
+2. **Testing:** Use open source and test apps to develop a proof of concept. Once complete, use our large user base to find users willing to test our prototype. Luckily, this feature had massive demand and there were users who were willing to try the earliest versions of it.
+
+The other two have more clever solutions. 
+
+### 1. Performance
+
+Not slowing down people's apps was core to our mission with session replay. 
+
+Our strategy to do wireframes was much less performance-intensive than others tools' reliance on screenshots. We still built screenshot mode, which provided a more accurate representation, but mostly focused on making our wireframe mode as good as possible. 
+
+![Screenshot vs Wireframe](https://res.cloudinary.com/dmukukwp6/image/upload/wireframe_78ce94bd4b.png)
+
+Many performance issues users faced were either caused by screenshot mode or unsupported data being captured. Both of these were solved by making wireframe mode better.
+
+On top of this, we try to offload as much work to our servers as possible. For example, the transformation to the rrweb schema happens on the server side. 
+
+### 2. Privacy
+
+As for privacy, we built the ability to mask all text inputs and images as well as redact certain views with `ph-no-capture` like this:
+
+<MultiLanguage>
+
+```xml
+<ImageView
+    android:id="@+id/imvProfilePhoto"
+    android:layout_width="200dp"
+    android:layout_height="200dp"
+    android:tag="ph-no-capture"
+/>
+```
+
+```swift
+let imvProfilePhoto = UIImageView(frame: CGRect(x: 50, y: 50, width: 100, height: 100))
+imvProfilePhoto.accessibilityIdentifier = "ph-no-capture"
+```
+
+</MultiLanguage>
+
+We added this functionality on both wireframe and screenshot mode. Although as nice as it would have been to have automatic masking like we do on the web, the ability to have masking at all enables privacy-sensitive teams to actually use replay. 
+
+## Making mobile session replay available for everyone
+
+Because mobile replay shares pricing with web replays, PostHog now offers the most generous free tier for mobile replays on the market. Where mobile replay was once a tool only available for large enterprise plans, now anyone can get 5000 replays for free every month.
+
+This enables us to help more developers have the tools they need to build successful products.
+
+## Further reading
+
+- [The 80/20 of early-stage startup analytics](/founders/early-stage-analytics)
+- [We decided to make session replay cheaper](/blog/session-replay-pricing)
+- [How to set up Android session replay](/tutorials/android-session-replay) and [How to set up iOS session replay](/tutorials/ios-session-replay)
```

Comment:
We can add the RN page

Reply:
Don't have that one yet 😅

Reply:
oh true that, we can link to https://posthog.com/docs/session-replay/react-native for now? but yeah its not a tutorial